service/history/queue/transfer_queue_processor_base.go

```diff
@@ -200,29 +210,34 @@ processorPumpLoop:
 		case <-t.shutdownCh:
 			break processorPumpLoop
 		case <-t.notifyCh:
-			if t.redispatchQueue.Len() <= t.options.MaxRedispatchQueueSize() {
-				t.processBatch()
-				continue
+			t.upsertPollTime(defaultProcessingQueueLevel, time.Time{})
```

Comment:
Is the defaultProcessingQueueLevel in this base?

Reply:
I don't quite understand your question, can you maybe elaborate? Are you asking where is the `defaultProcessingQueueLevel` define? Right now it's defined in `transfer_queue_processor.go`, I don't think that's a good place, I will move it to `queue_processor_util.go`